### PR TITLE
Backport changes from Cortx-v1.0-Beta branch

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise yashodhan.pise@seagate.com

### CODE REVIEW

**Review by 83bhp — CHANGES_REQUESTED** [review]
Hold on to this patch there are some changes required

File: cli/src/factory_ops/unboxing/config_update [hunk 1]
```diff
@@ -224,6 +224,215 @@ function update_cluster_sls {
...
+                echo "Could not find the target_build line number in $_release_sls_path_user, ignoring" >> $LOG_FILE
+            fi
+        fi
+    fi
+
+    echo "Refreshing Salt pillars" >> ${LOG_FILE}
+    salt '*' saltutil.refresh_pillar ${salt_opts}
+    echo "Checking if the target_bulid url is removed from release pillar" >> ${LOG_FILE}
+
+    _node1_tgt_build=$(salt srvnode-1 pillar.get eos_release:target_build --output=newline_values_only ${salt_opts})
```
Comment (83bhp) [inline cli/src/factory_ops/unboxing/config_update:420]: release pillar name needs to be changed.

File: cli/src/factory_ops/unboxing/config_update [hunk 2]
```diff
@@ -224,6 +224,215 @@ function update_cluster_sls {
...
+    _node2_tgt_build=$(salt srvnode-2 pillar.get eos_release:target_build --output=newline_values_only ${salt_opts})
```
Comment (83bhp) [inline cli/src/factory_ops/unboxing/config_update:421]: same here.

File: cli/src/factory_ops/unboxing/config_update [hunk 3]
```diff
@@ -224,6 +224,215 @@ function update_cluster_sls {
...
+                yes | cp -f ${_release_sls_path}.bkp $_release_sls_path
+            else
+                echo "Successfully updated target_build in $_release_sls_path" >> ${LOG_FILE}
+            fi
+        else
+            echo "Could not find the target_build line number in $_release_sls_path, ignoring" >> $LOG_FILE
+        fi
+    fi
+
+    if [[ -f "${PRVSNR_ROOT}/pillar/user/groups/all/release.sls" ]]; then
```
Comment (83bhp) [inline cli/src/factory_ops/unboxing/config_update:392]: this path won't work after new deployment method.

**Review by 83bhp — CHANGES_REQUESTED** [review]

File: cli/src/factory_ops/unboxing/config_update (hunk 1 above)
Comment (ypise) [reply cli/src/factory_ops/unboxing/config_update]: Addressed

File: cli/src/factory_ops/unboxing/config_update (hunk 3 above)
Comment (ypise) [reply cli/src/factory_ops/unboxing/config_update]: addressed

File: cli/src/factory_ops/unboxing/config_update (hunk 2 above)
Comment (ypise) [reply cli/src/factory_ops/unboxing/config_update]: addressed

**Comment by ypise** [comment]
Request has been addressed with other PRs

**Pull request closed without merging**